DataFormats/BeamSpot/src/classes_def.xml

```diff
@@ -5,11 +5,17 @@
   </class>
   <class name="edm::Wrapper<reco::BeamSpot>"/>
 
+  <class name="BeamSpotHost"/>
+  <!-- BeamSpotHost::Product must be listed before the aliased-to type -->
+  <!-- TODO: we should find a better way than replicating the class versions and checksums -->
+  <class name="BeamSpotHost::Product" ClassVersion="3">
+   <version ClassVersion="3" checksum="280341519"/>
+  </class>
+
   <class name="BeamSpotPOD" ClassVersion="3">
    <version ClassVersion="3" checksum="280341519"/>
   </class>
```

Comment:
@pcanal I think we discussed about the apparent need to define the class versions and checksums twice (the `BeamSpotHost::Product` is an alias for `BeamSpotPOD`). Or, I didn't manage to come up with any other way that would result in both type names to end up in the `.rootmap` file. I wouldn't hold this PR for an improvement, but we should follow up in https://github.com/cms-sw/cmssw/issues/49458.